dis_snek/models/discord_objects/user.py

```diff
@@ -328,6 +328,19 @@ async def remove_role(self, role: Union[Snowflake_Type, Role], reason: str = MIS
             role = role.id
         return await self._client.http.remove_guild_member_role(self._guild_id, self.id, role, reason=reason)
 
+    async def has_role(self, *roles: Union[Snowflake_Type, Role]) -> bool:
+        """
+        Checks if the user has the given role(s)
+        Args:
+            roles: The role(s) to check whether the user has it.
+        """
+        for role in roles:
+            if isinstance(role, Role):
+                role = role.id
+            if str(role) not in self._role_ids:
```

Comment:
Why convert to str here? role id should be an integer since it's a snowflake. (unless we didn't convert properly somewhere prior)

Reply:
I also did think it was int first and ran my checks and it failed, but then i checked the `_role_ids` it was str

Reply:
yea forgot to convert, just fixed it in latest commit should be all int now.

Reply:
@benwoo1110  Yes I think conversion isn't proper, `missing the to_snowflake converter`, I added ` converter=lambda ids:[to_snowflake(id) for id in ids]` (Is there any other converter? the list_converter didn't work) and its all ints in the __role_ids

Reply:
Hmm I get all ints after commit https://github.com/LordOfPolls/dis_snek/commit/06b8995a7e9abf144bbeeddfb3dc7d877110d18b Either you haven't update your branch or something you are doing different.

Reply:
Oh, that commit came after I committed, so i didn't know